03.macos/05.solving-problems/09.advanced-settings/docs.en.md

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Advanced Settings guide'
+taxonomy:
+    category:
+        - docs
+visible: true
+---
+
+## How to reach the Advanced settings
+
+> Changing *Low-level settings* can cause problems with the performance of AdGuard, may break the Internet connection or compromise your security and privacy. You should only open this section if you are sure of what you are doing or our support-team has asked you about it.  
+
+To go to *Advanced settings*, follow this route: the AdGuard icon at the tray menu → the gear icon → Advanced → Advanced settings.
+
+## Advanced settings
+
+`network.extension.exclude.domains` 
+
+The listed domains will be excluded from filtering in *Network Extension* mode. Use a comma or line break to separate values. 
+
+`network.extension.exclude.ports`
+
+The listed ports will be excluded from filtering in *Network Extension* mode. Use a comma or line break to separate values.
+
+`network.extension.route.exclude` 
+
+The listed routes will be excluded from filtering in *Network Extension* mode. Set routes using an IP address or destination CIDR. Separate values with commas or line breaks. 
+
+`network.extension.http.downgrade.bundleids`
+
+Here you can specify a list of applications for which the HTTP/2 protocol will be downgraded to HTTP/1.1 in the *Network Extension* filtering mode. The bundle ID should be separated by a comma or a line break.
+
+`network.extension.monterey.force.split.tunnel` 
+
+Here you can prohibit AdGuard from using the "default route" which is enabled by default in *Network Extension* mode on macOS Monterey. AdGuard uses "default route" to disable iCloud Private Relay and Protect Mail Activity, because it cannot work with them simultaneously. 
+
+You will find more information about the problem [in this article](https://kb.adguard.com/en/macos/solving-problems/icloud-private-relay). 
+
+`network.extension.dns.redirect.exclude.bundleids` 
+
+Here you can specify a list of applications that make DNS queries directly not via system DNS resolver (for example, some VPN clients or DNS filtering applications). DNS filtering will be disabled for them in the *Network Extension mode*. The bundle IDs should be separated by a comma or line break.
+
+`network.https.ocsp.check` 
+
+By setting `true`, you enable HTTPS certificate revocation checking.
+
+`network.filtering.localnetwork` 
+
+By setting `true`, you enable local network filtering. 
+
+`network.filtering.localhost` 
+
+By setting `true`, you enable LoopBack filtering. 
+
+`upstream.proxy` 
+
+In this string, you can specify a proxy server for upstream connections.
```

Comment:
We need a couple of examples here:
1. One for a proxy that requires authentication
2. One for a proxy that does not require it